app/uk/gov/hmrc/agentclientrelationships/connectors/EnrolmentStoreProxyConnector.scala

```diff
@@ -67,7 +67,8 @@ class EnrolmentStoreProxyConnector @Inject()(
         else {
           if (userIds.lengthCompare(1) > 0)
```

Comment:
is this check `userIds.lengthCompare(1) > 0 `  useful here except we are just doing a warn logging?

Reply:
je ne sais pas